tests/type-destructors/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -69,6 +69,7 @@ var x1: $NonMaybeType<number | string> = true;
 // err, boolean ~> number|string
 var x2: $PropertyType<{ p: number } | { p: string }, \"p\"> = 0;
 // ok, number ~> number|string
-var x3: $PropertyType<{ p: number } |
-  { p: string }, \"p\"> = true; // err, boolean ~> number|string"
+var x3: $PropertyType<
```

Comment:
This is kind of funky... Looks like the trailing comment is throwing it off, but it brings up a point. Do we want the same line-breaking behavior when a union type is used as a type parameter?

```js
var x: TypeWithParameters<{ p: 'long-test-string' } | { p: 'other-long-test-string' } | { p: 'one-more' }>; 
```